### PR TITLE
update: Add ATB to popup too

### PR DESCRIPTION
**Pull request by mrshu**
* Add ATB to popup too.

Signed-off-by: mr.Shu <mr@shu.io>

### CODE REVIEW

**Comment by bsstoner** [comment]
👍

File: duckduckgo.safariextension/Info.plist [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.7.2</string>
+	<string>1.7.3</string>
 	<key>CFBundleVersion</key>
 	<string>32</string>
```
Comment (mrshu) [inline duckduckgo.safariextension/Info.plist]: @bsstoner unfortunately, this needs to get updated too =(

File: duckduckgo.plist [hunk 2]
```diff
@@ -12,7 +12,7 @@
        <key>CFBundleVersion</key>
        <string>32</string>
```
Comment (mrshu) [inline duckduckgo.plist]: @bsstoner same as below

File: duckduckgo.safariextension/Info.plist (hunk 1 above)
Comment (bsstoner) [inline duckduckgo.safariextension/Info.plist]: the CFBundleVersion? Increment to 33?

**Pull request merged**